playground/src/PlaygroundView.css

```diff
@@ -0,0 +1,12 @@
+.playground-main-row {
```

Comment:
We usually use `.module.scss` files, with class names that contain a hash.

Can you convert this file to `.module.scss` and change these class names to something that can be a TS variable name (like `playgroundMainRow`)?

Reply:
So name the file soemthing like `playgroundView.module.scss`? I looked on the odsp web repo and there were examples of `.module.scss` being used like that

Reply:
Yeah, exactly.